jme3-core/src/main/java/com/jme3/shadow/AbstractShadowFilter.java

```diff
@@ -93,18 +88,12 @@ protected boolean isRequiresDepthTexture() {
         return true;
     }
 
-    public Material getShadowMaterial() {       
```

Comment:
I see you removed this redundant method (I assume because there is already a getMaterial() method that does the same thing)

However, it may be best to deprecate this first, and then maybe remove it later (if at all).

I know that having a duplicate method may feel dirty, but now that it has been there so long, it can be better to leave it so that existing apps that are using this method don't break with the new version.

Reply:
I agree, I restore the original method with the ‘deprecated’ annotation and add the javadoc